src/sass/components/dsf-header.scss

```diff
@@ -0,0 +1,29 @@
+// -----------------------------------------------------------------------------
+// This file contains all styles related to the header of the site/application.
+// -----------------------------------------------------------------------------
+//$dsf-primary: #31576F;
+$dsf-white: #ffffff;
+@import "../../../src/sass/base/dsf-variables.scss";
+.navbar {
+    background-color: $dsf-white !important;
+   .container-fluid {
+        background-color: $dsf-primary;
+      img {
```

Comment:
Can we put the base64 encoded string as a variable somewhere? To make the file easier to read and then we can reuse it too.